### ISSUE TITLE
map svg images lack alt text

### ISSUE THREAD

**Issue by lfrancis** [forumone/peacecorps-site#399]
IMG tags must have an ALT attribute. Section 508 1194.22 (a) WCAG 2.0 A F65  

 Add an ALT attribute describing each image, which screen readers read aloud. Spacer images and purely decorative images should use ALT=''. •NVDA 2015.2 FF40 Windows 7 Reading: Image ignored.
•JAWS 16.0.4350 FF40 Windows 7 Reading: Image ignored.
•WindowEyes 8.4 IE11 Windows 7 Reading: Image ignored.
•VoiceOver Safari OSX 10.10 Reading: Image filename read out.
•VoiceOver Safari iOS 7.1 Touch: Image filename read out.
•SaToGo 3.4.96.0 IE11 Windows 7 Reading: Image ignored.

 https://stage-donate.peacecorps.gov/donate/projects-funds/
